operations/patch_util.go

```diff
@@ -205,6 +212,28 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 	return
 }
 
+// Sets the patch's alias to either the passed in option or the default
+func loadAlias(p *patchParams, conf *ClientSettings) (err error) {
```

Comment:
we have tended to avoid the pattern of declaring variables in the return signature, as it's really easy to do weird things. 

it does tend to make things a bit more verbose (but also more clear)